### DISCUSSION TITLE
pnpm dlx @antfu/eslint-config@latest 无法选择

### DISCUSSION THREAD

**Discussion by Liu-code3** [antfu/eslint-config#587] [Q&A]
### 当我使用方向键来选择react的时候 然后按下回车键去确定的时候 控制台给我的信息为none 
![image](https://github.com/user-attachments/assets/9072c53c-613c-4772-bb39-e0a46a19e812)

**Comment by BabyLy233** [comment]
I encountered the same situation. At the same time my `pnpm dlx` was not working, so I used `npx`
Maybe the same issue #597 
![QQ_1725601864311](https://github.com/user-attachments/assets/98de8466-c3cf-43d5-97c9-26742a729264)

**Reply by BabyLy233** [reply]
I used this in a new project created by `create-vue`

**Reply by Liu-code3** [reply]
Is this a bug in Antfu scaffolding? My solution is to manually download these ESLint libraries
